draft-ietf-ppm-dap.md

```diff
@@ -1487,9 +1495,8 @@ If a report's ID matches that of a previously uploaded report, the Leader MUST
 ignore it. In addition, it MAY alert the Client with error `reportRejected`.
 
 The Leader MUST ignore any report pertaining to a batch that has already been
-collected (see {{input-share-validation}} for details). Otherwise, comparing
-the aggregate result to the previous aggregate result may result in a privacy
```

Comment:
Why remove "Otherwise, ..."? This seems like a useful clarification.

Reply:
Because I think that's now well explained in {#replay-protection}, so no need to repeat it here.